### DISCUSSION TITLE
Welcome to jess Discussions!

### DISCUSSION THREAD

**Discussion by matthew-dean** [jesscss/jess#5] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kevinramharak** [comment]
I have been semi-following your work since I had to discover LESS when becoming a front-end developer at my current company. Haven't had that much to do with LESS once we migrated to bootstrap 4. I remembered a Less issue where you said you were working on something new instead of rewriting LESS in typescript. This looks very promising, combining CSS with the actual JS runtime could pave the way for great things.

I will keep my eye on this repo, hoping I can find some time to contribute to the LanguageServerProtocol. Been researching the typescript compiler and its compiler api / language server plugins so hoping that will pay off in a project like this.

Keep up the good work.

**Reply by matthew-dean** [reply]
Great!

**Reply by kevinramharak** [reply]
@matthew-dean I noticed you picked this up again, my inbox is getting spammed with closed issues and PR's :). Good to see you still want to make this work. I might not be able to contribute much at the current moment, but I wanted to at least mention I noticed your effort and I appreciate it.
